### PR TITLE
OSDOCS-7749: Follow-up edits

### PR DESCRIPTION
**Pull request by bmcelvee**
Version(s):
4.13+

Issue:
https://issues.redhat.com/browse/OSDOCS-7749

Link to docs preview:
https://65124--docspreview.netlify.app/openshift-rosa/latest/rosa_planning/rosa-cloud-expert-prereq-checklist
https://65124--docspreview.netlify.app/openshift-rosa/latest/rosa_planning/rosa-sts-aws-prereqs

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by bmcelvee** [comment]
@michaelryanmcneill @laubai FYI

**Comment by bmcelvee** [comment]
Sifting through and working to resolve build errors to get this update in.

File: rosa_planning/rosa-cloud-expert-prereq-checklist.adoc [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+----
+
+=== Red Hat account
+
+* Create a https://console.redhat.com/[{hybrid-console}]  account if you have not already.
+
+=== ROSA CLI (`rosa`)
+
+. Enable ROSA from your AWS account on the https://console.aws.amazon.com/rosa/[AWS console] if you have not already.
+. Install the CLI from xref:../rosa_install_access_delete_clusters/rosa_getting_started_iam/rosa-installing-rosa.html[Installing the Red Hat OpenShift Service on AWS (ROSA) CLI, rosa[] or from the OpenShift console https://console.redhat.com/openshift/downloads#tool-rosa[AWS console].
```
Comment (EricPonvelle) [inline rosa_planning/rosa-cloud-expert-prereq-checklist.adoc:92]: Renders as 
image


Also, I'm not sure how line 91 works as a link. It's just a URL attached to the hyperlinked text, but I thought you needed `link:`.

**Review by EricPonvelle — COMMENTED** [review]
You could also go into this module and change the string to code block, but that's the only other thing I caught:

image

**Pull request merged**